File: Python/underautomation/universal_robots/ssh/tools/common/shell_data_event_args.py
import typing
import clr
import os
clr.AddReference(os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "..", "..",  'lib', 'UnderAutomation.UniversalRobots.dll')))
from UnderAutomation.UniversalRobots.Ssh.Tools.Common import ShellDataEventArgs as shell_data_event_args

class ShellDataEventArgs:
	def __init__(self, data: typing.List[int], _internal = 0):
		if(_internal == 0):
			self._instance = shell_data_event_args(data)
		else:
			self._instance = _internal
	@property
	def data(self) -> typing.List[int]:
		return self._instance.Data
	@property
	def line(self) -> str:
		return self._instance.Line

File: UnderAutomation.UniversalRobots.ObfuscatedSources/Ssh/Tools/Common/ShellDataEventArgs.cs
//              WARNING
// This file is an empty shell containing only public C# items.
// The internal code is hidden; to access it, you need to obtain a Source licence of the library.


namespace Ssh.Tools.Common {
	/// <summary>
	/// Provides data for Shell DataReceived event
	/// </summary>
	public class ShellDataEventArgs : EventArgs {

		/// <summary>
		/// Gets the data.
		/// </summary>
		public byte[] Data { get; }

		/// <summary>
		/// Gets the line data.
		/// </summary>
		public string Line { get; }

		/// <summary>
		/// Initializes a new instance of the <xref href="UnderAutomation.UniversalRobots.Ssh.Tools.Common.ShellDataEventArgs" data-throw-if-not-resolved="false"></xref> class.
		/// </summary>
		/// <param name="data">The data.</param>
		public ShellDataEventArgs(byte[] data)
		{
			// Source is hidden, a Source licence is needed to access internal code...
		}

		/// <summary>
		/// Initializes a new instance of the <xref href="UnderAutomation.UniversalRobots.Ssh.Tools.Common.ShellDataEventArgs" data-throw-if-not-resolved="false"></xref> class.
		/// </summary>
		/// <param name="line">The line.</param>
		public ShellDataEventArgs(string line)
		{
			// Source is hidden, a Source licence is needed to access internal code...
		}
	}
}
